umm-spec-lib/src/cmr/umm_spec/test/dif10_expected_conversion.clj

```diff
@@ -336,10 +337,25 @@
   (when-let [media (first media)]
     [media]))
 
+(defn- expected-dist-parsed-format
+  "I could not figure out how to not create this function using the update call in the expand-file-dist-infos function below. So this function exists to replace the new value with the old."
```

Comment:
You have a bunch of docstrings that are just one really long line and hard to read - could you break them into multi-line strings? It'll make them easier to read

Reply:
I have updated the doc strings.